_data/ph_authors.yml

```diff
@@ -971,3 +971,18 @@
       en: |
           Jeff Blackadar is a part-time undergraduate history student at Carleton University. His career is in the field of information technology.
 
+          
+- name: Sofia Papastamkou
+  email: spapastamkou@gmail.com
+  twitter: s_papastamkou
+  github: spapastamkou
+  team: true
+  team_start: 2018
+  institution: CNRS
+  sortname: Papastamkou
+  affiliation: Maison européenne des sciences de l'homme et de la société Lille Nord de France
+  bio:
+        en: | 
+        Sofia Papastamkou is a historian and Research Engineer in Digital Humanities at the French National Center for Scientific Research (CNRS). 
```

Comment:
A small change is required: you must indent all lines after `en: |` so that they line up with the `|`